src/main/java/appeng/core/Registration.java

```diff
@@ -216,7 +214,6 @@ private void registerCraftHandlers( final IRecipeHandlerRegistry registry )
 		registry.addNewCraftHandler( "mekechamber", MekEnrichment.class );
 		registry.addNewCraftHandler( "grind", Grind.class );
 		registry.addNewCraftHandler( "crusher", Crusher.class );
-		registry.addNewCraftHandler( "grindfz", GrindFZ.class );
```

Comment:
Removing the handler will cause the recipe system to log error about unsupported types. Nothing breaking, but might be relevant shoul we move them to their integration module (even register them as stub or similar)